plugins/catalog-backend-module-bitbucket-cloud/src/providers/BitbucketCloudEntityProvider.test.ts

```diff
@@ -505,6 +518,19 @@ describe('BitbucketCloudEntityProvider', () => {
     })[0];
 
     server.use(
+      rest.get(
+        `https://api.bitbucket.org/2.0/workspaces/test-ws/projects`,
+        (_req, res, ctx) => {
+          const response = {
+            values: [
+              {
+                key: 'TEST',
```

Comment:
It would be great to adjust the tests to test with more than one project. Currently, it's still the same besides the additional call for the projects. The test would still work if you only keep that call and revert the rest.

At the search call, we also verify the query
```
if (!query || !query.includes('repo:test-repo'))
```